docs/UserGuide.md

```diff
@@ -138,7 +147,7 @@ Examples:
 
 ### Clearing all entries : `clear`
 
-Clears all entries from the address book.
+Clears all entries in the current list view from VirusTracker.
```

Comment:
Might consider changing to 'Clears all entries from Virus Tracker' to match description in line 42.